src/App.tsx

```diff
@@ -1,35 +1,36 @@
 import React, { useRef, useEffect } from 'react'
-import { Provider } from 'react-redux'
+
+import { PortalProvider } from '@gorhom/portal'
 import { Platform } from 'react-native'
 import { SafeAreaProvider } from 'react-native-safe-area-context'
-import { PortalProvider } from '@gorhom/portal'
-
-import createStore from './store'
-import WebApp from './components/web/WebApp'
-import Audio from './components/audio/Audio'
-import GoogleCast from './components/audio/GoogleCast'
-import OAuth from './components/oauth/OAuth'
 import WebView from 'react-native-webview'
-import PushNotifications from './notifications'
-import { setup as setupAnalytics } from './utils/analytics'
-import useConnectivity from './components/web/useConnectivity'
-import { incrementSessionCount } from './hooks/useSessionCount'
-import Notifications from './components/notifications/Notifications'
-import Search from './components/search/Search'
-import SignOnNav from './components/signon/NavigationStack'
-import { WebRefContextProvider } from './components/web/WebRef'
-import BottomBar from './components/bottom-bar'
-import MobileUploadDrawer from './components/mobile-upload-drawer'
-import EnablePushNotificationsDrawer from './components/enable-push-notifications-drawer'
-import CollectibleDetailsDrawer from './components/collectible-details-drawer'
+import { Provider } from 'react-redux'
+
+import Audio from 'app/components/audio/Audio'
+import GoogleCast from 'app/components/audio/GoogleCast'
+// import BottomBar from 'app/components/bottom-bar'
```

Comment:
commented out

Reply:
It's still supposed to be commented out bc we don't want to show the RN BottomBar yet